common/lib/api-client/middleware/response.js

```diff
@@ -0,0 +1,48 @@
+const { isArray } = require('lodash')
+
+function needsDeserialization(method) {
+  return ['GET', 'PATCH', 'POST'].indexOf(method.toUpperCase()) !== -1
+}
+
+module.exports = {
+  name: 'app-response',
+  res: function (payload) {
+    /*
+     * Note: This has been overridden from the devour library to support the
+     * use of `Got` instead of `axios` to make requests
+     */
+    const { jsonApi, res, req } = payload
+    const { data, errors, meta, links, included } = res.body || {}
```

Comment:
I assume it's this line that means we need to override the response middleware, replacing data with body?

Reply:
Yep. At this stage there isn't a huge deal more, however, this will be extended once we start looking at moving pagination inside the API client. I have a [wip branch](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/tree/got-pagination) that shows that implementation.